src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -48,21 +54,31 @@ const App = () => (
 );
 
 const MyChild = () => {
-  const { tileId, isVideoEnabled, setIsVideoEnabled, toggleVideo } = useLocalVideo();
+  const { tileId, isVideoEnabled, setIsVideoEnabled, 
+  isReachVideoLimit, setIsReachVideoLimit, toggleVideo } = useLocalVideo();
 
   return (
     <p>Tile ID: {tileId}</p>
 
     <p>
-      {setIsVideoEnabled ? 'LocalVideo is enabled' : 'LocalVideo is disabled'}
+      {isVideoEnabled ? 'LocalVideo is enabled' : 'LocalVideo is disabled'}
+    </p>
+
+    <p>
+      {isReachVideoLimit ? 'Video limit is reached' : 'Video limit is not reached'}
     </p>
 
     <button onClick={() => setIsVideoEnabled(true)}>
       Set isVidelEnabled to true
     </button>
+
+    <button onClick={() => setIsReachVideoLimit(true)}>
+      Set isReachVideoLimit to true
+    </button>
     
     <button onClick={toggleVideo}>
-      {isVideoEnabled ? 'Stop your video' : 'Start your video'}
+      {isVideoEnabled ? 'Stop your video' : isReachVideoLimit ? 
+      'Reach the video limit, can not turn on video' : 'Start your video'}
```

Comment:
Wrong tense like the variable, should be something like 'Has reached the video limit, can not turn on video'

Reply:
Updated